tower_eval/models/anthropic/generator.py

```diff
@@ -73,14 +73,9 @@ def _generate(self, input_line: str) -> str:
             retry_max_interval=self.retry_max_interval,
         )
 
-        response = response.content[0].text.split("\n\n")[
-            -1
-        ]  # hacky way to remove the agent's preamble in the response
+        response = response.content[0].text
```

Comment:
don't we need to keep the previous solution? has the behaviour changed since then?

Reply:
it was a hack I did once, but I forgot to remove. The model wasn't responding well to the prompt we were using. Changing the prompt, or using a good system prompt removes the need for this.